packages/tasks/src/tasks/chat-completion/spec/input.json

```diff
@@ -47,17 +51,42 @@
 			"type": "number",
 			"description": "The value used to modulate the logits distribution."
 		},
+		"tools": {
```

Comment:
Should we also update `output.json` to include `output.choices[].message.tool_calls` or not yet supported by TGI?

Reply:
Same comment for `output.choices[].delta.tool_calls` in `output_stream.json`?